dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -102,15 +103,14 @@ public void testArtifacts()
             "Class com.jcraft.jzlib.JZlib is not found;\n"
                 + "  referenced by 4 class files\n"
                 + "    io.grpc.netty.shaded.io.netty.handler.codec.spdy.SpdyHeaderBlockJZlibEncoder"
-                + " (io.grpc:grpc-netty-shaded:1.13.1)");
+                + " (io.grpc:grpc-netty-shaded:1.30.2)");
```

Comment:
Diff:

Linkage Errors before the change (with 0.65.0-beta): https://gist.github.com/suztomo/b0a19b75b2a79960ab4a978b4ba70c8d
Linkage Errors after the change (with 1.35.2): https://gist.github.com/suztomo/9a44b2d6994200fd5fe3110830e0e01b